dockers/docker-sonic-restapi/supervisord.conf

```diff
@@ -3,10 +3,26 @@ logfile_maxbytes=1MB
 logfile_backups=2
 nodaemon=true
 
-[program:restapi]
-command=/usr/sbin/go-server-server -loglevel trace
+[program:start.sh]
+command=/usr/bin/start.sh
 priority=1
 autostart=true
+autorestart=true
+stdout_logfile=syslog
+stderr_logfile=syslog
+
+[program:restapi]
+command=/usr/sbin/go-server-server -enablehttps=true -clientcert=/etc/sonic/certificates/restapiclient.crt -servercert=/etc/sonic/certificates/restapiserver.crt -serverkey=/etc/sonic/certificates/restapiserver.key -loglevel=trace
```

Comment:
should look for this as example. 

https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-sonic-telemetry/telemetry.sh

we should read the certs form config db.